FalmouthFalcons/iteration-3
# Categories showing with product quantity and inventory

Description:
Categories showing with product quantity and inventory

Number of Fixes:
-Ticket 4

Related Ticket(s):
- Ticket 4

Problem to Solve:
- Categories showing with product quantity and inventory

Proposed Changes:
-Categories showing with product quantity and inventory

Expected Behavior:
- Categories showing with product quantity and inventory

Steps to Test Solution
1) rails db:migrate
2) rails db:seed
3) git fetch, git checkout ticket4/categories

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
